controllers/searchoperator_controller.go

```diff
@@ -380,150 +393,165 @@ func int32Ptr(i int32) *int32 { return &i }
 
 func int64Ptr(i int64) *int64 { return &i }
 
+func compareLabels(metadataLabels, ssetLabels map[string]string) bool {
```

Comment:
Comment for the purpose of the function would be nice .

Reply:
Done